src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -2686,6 +2682,25 @@ private void SqueezeDatabasesToSingleNodeCluster(ClusterOperationContext context
             ExecuteManyOnDispose(context, index, type, tasks);
         }
 
+        private static void Shrink(string oldTag, string newTag, DatabaseTopology topology, RawDatabaseRecord rawRecord, ref List<DatabaseRecord> toDelete, ref List<DatabaseRecord> toShrink)
```

Comment:
Shrink what?

Reply:
I guess `SqueezeDatabasesToSingleNodeCluster` method was throwing while I tested shared subscription cluster test, anyway I reverted the code.